### PR TITLE
Added Polkadot Poland DAO - PolskaDOT

### PR DESCRIPTION
**Pull request by PolkadotPolandDAO-PolskaDOT**

### CODE REVIEW

File: registry/polkadot.json [hunk 1]
```diff
@@ -396,5 +396,13 @@
         "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/jeeper_delegate.png",
         "shortDescription": "Governance Researcher @opengov.watch",
         "isOrganization": false
+    },
+    {
+        "address": "1313ciB4VzPeH3n1QKJym1brBmzRdfHBEctWipgH4uGsyF6n",
+        "name": "Polkadot Poland DAO - PolskaDOT",
+        "image": "https://raw.githubusercontent.com/PolkadotPolandDAO-PolskaDOT/opengov-delegate-registry/66feb88945768f4b75bf467e2c4fb3c8e9788b37/images/Polkadot%20Poland%20DAO%20-%20PolskaDOT.png",
```
Comment (stepanLav) [inline registry/polkadot.json:403]: Hello!
based our rules, all links should reach to our repository, please change it to /novasamatech/opengov-delegate-registry/

**Comment by stepanLav** [comment]
Could you, also, provide proof of authority for your account? That process described in our wiki:
https://docs.novawallet.io/nova-wallet-wiki/opengov/delegate-registry

File: registry/polkadot.json (hunk 1 above)
Comment (PolkadotPolandDAO-PolskaDOT) [reply registry/polkadot.json]: Links corrected

**Comment by PolkadotPolandDAO-PolskaDOT** [comment]
Here comes the confirmation system:remark extrinsic link:

https://polkadot.subscan.io/extrinsic/22532514-2

**Comment by PolkadotPolandDAO-PolskaDOT** [comment]
https://polkadot.subscan.io/multisig_extrinsic/22532514-2?call_hash=0xf8adeb3562998de5139a9c24090976412e429e145c021a76026864f56bb52818

**Review by stepanLav — APPROVED** [review]

**Pull request merged**